code/modules/jobs/job_types/security.dm

```diff
@@ -51,7 +51,7 @@ Head of Security
 	suit_store = /obj/item/gun/energy/e_gun
 	r_pocket = /obj/item/device/assembly/flash/handheld
 	l_pocket = /obj/item/restraints/handcuffs
-	backpack_contents = list(/obj/item/melee/baton/loaded=1)
+	backpack_contents = list(/obj/item/melee/baton/loaded=1, /obj/item/gun/energy/e_gun/mini = 1)

```

Comment:
giving the man with full access to the armory, a spare energy gun, and his own personalized weapon another gun.

Reply:
he should only have the sword the nobility don't use poor man's weaponry.